### ISSUE TITLE
The splash block doesn't have it's text shown

### ISSUE THREAD

**Issue by brianlee389** [thomaspark/snowball#170]
Here is the image:
![zindex](https://cloud.githubusercontent.com/assets/6540823/9742124/5e4cf19a-562c-11e5-8b6e-3ecca96ca3b4.png)
I tried changing the z-index of the background div to -1 in the inspect element and the text was shown.

**Comment by thomaspark** [comment]
Couple of things to look out for:
- `background-attachment: fixed` (which is used by parallax) is buggy in Chrome (see [1](http://stackoverflow.com/questions/23571578/chrome-issue-with-background-attachment-fixed-and-position-fixed-elements), [2](http://stackoverflow.com/questions/20268962/fixed-attachment-background-image-flicker-disappear-in-chrome-when-coupled-with), [3](http://stackoverflow.com/questions/19258739/absolute-positioned-html5-video-element-with-negative-z-index-breaks-background?rq=1)). One workaround for this related to setting a `z-index`. Not sure what impact of setting `z-index: -1` of background div is, but something we should test.
- Certain themes have an article background color. Setting `z-index: -1` might obscure the splash block's background in this case.
- Does setting the content div z-index to a higher value also fix the problem? If so, we could test that to see if it deals better with parallax in Chrome and other themes with background.

**Comment by brianlee389** [comment]
I went through the stackoverflow for chrome, wasn't exactly sure how it will affect the solution I just made, which is adding these properties to the css classes in splash/template.html:
.snowball-block-splash {
z-index:0;
}
.snowball-block-splash-bg {
z-index:-1;
}

I haven't tried this with many other themes, but I tried it with parallax and Safari, Chrome, and FF and it worked. I don't know what problems would occur from this.
I haven't tested with IE because I forgot the password for my test account.
